### ISSUE TITLE
reconnect with new user id failed

### ISSUE THREAD

**Issue by yujinlim** [layerhq/layer-websdk#101]
```
angular.js:13920 TypeError: callback is not a function
    at db-manager.js:962
    at DbManager.onOpen (db-manager.js:149)
    at DbManager.deleteTables (db-manager.js:961)
    at Client._clearStoredData (client-authenticator.js:612)
    at Client.connect (client-authenticator.js:231)
```

from what i can see, is that `this._clearStoredData();` is expecting a callback. This is on version 1.1

**Comment by michaelkantor** [comment]
Ok; I've created a branch with a fix to this.  But investigating how well the layer.Client supports CHANGING User IDs is a separate task which I'll need to plan on for next week.

If you can, call `client.destroy()` and start a new client for the new user.

I expect this branch should be merged to master late next week after any other issues are reported.

Thanks for identifying this use case and error!

Michael

**Comment by yujinlim** [comment]
@michaelkantor thanks

**Comment by michaelkantor** [comment]
Fixes to `deleteTables()` is now in 2.0 as well as 3.0.0-beta and are released.
